tools/development-error-app/development-error-app.js

```diff
@@ -0,0 +1,55 @@
+if (Meteor.isServer) {
+  // Set connection to null to use Minimongo on server
+  Errors = new Mongo.Collection("errors", {connection: null});
+  Refreshing = new Mongo.Collection("refreshing", {connection: null});
+
+  Meteor.publish("errors", function () {
+    return Errors.find();
+  });
+
+  Meteor.publish("refreshing", function () {
+    return Refreshing.find();
+  });
+}
+
+if (Meteor.isClient) {
+  // Need to define connection twice because setting connection to null on
+  // client means it should ignore the DDP connection to the server
+  Errors = new Mongo.Collection("errors");
+  Refreshing = new Mongo.Collection("refreshing");
+  Meteor.subscribe("errors");
+  Meteor.subscribe("refreshing");
+  Template.body.helpers({
+    errors: function () {
+      return Errors.find({}, {sort: {createdAt: -1} });
+    },
+    refreshing: function () {
+      return Refreshing.find({_id: 'is-app-refreshing'}).fetch()[0].value;
+    }
+  });
+}
+
+Meteor.methods({
+  addErrorMessage: function (errMsg) {
+    if (this.isSimulation)
```

Comment:
You could just define these methods inside the server code to avoid them generating simulations.